.github/workflows/merge-announce.yml

```diff
@@ -0,0 +1,20 @@
+name: push
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  announcepush:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get repo
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Send push to Discord
+        run: |
+          curl -i -H "Accept: application/json" -H "Content-Type: application/json" -X POST --data \
```

Comment:
I think its better to use gh tool when running in actions.

Reply:
?

Reply:
Oh, I see. This is push to discord ;)